src/App.js

```diff
@@ -19,64 +20,47 @@ class App extends Component {
         text: 'FPO image',
       },
       { type: BlockType.Blockquote, text: 'this is a quote' },
-    ].map(this.toObjectsWithKeys),
+    ].map(this.toObjectsWithKey),
   };
 
-  toObjectsWithKeys(block, index) {
+  toObjectsWithKey(block, index) {
```

Comment:
Let's call this `toObjectWithKey` (`Object` singular, not `Objects`) … sorry, not sorry

Reply:
This is done @mysterycommand if you wanted to double check it.